samples/nrf9160/ciphersuites/README.rst

```diff
@@ -0,0 +1,147 @@
+.. _ciphersuites:
+
+nRF9160: TLS Cipher Suites
+#####################
```

Comment:
this is failing the doc build. 
`Warning, treated as error:
/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/samples/nrf9160/ciphersuites/README.rst:4:Title underline too short.`